app/components/shared/Menu/button.js

```diff
@@ -24,7 +24,7 @@ class Button extends React.Component {
         link={this.props.link}
       >
         <div className="flex">
-          <div className="flex-auto">{this.props.children}</div>
+          <div className="flex-auto truncate">{this.props.children}</div>
```

Comment:
oh right, there’s still a truncate on `BaseButton`, which isn’t doing anything - remove that!